### DISCUSSION TITLE
polygon hard fork 17th January 2023 - Will it affect Edge in any way?

### DISCUSSION THREAD

**Discussion by ghost** [0xPolygon/polygon-edge#1128] [General]
As the title says, will the hard fork affect Edge in anyway? We are on 0.6.1

Thanks

**Comment by laviniat1996** [comment]
Hi @Nexsuperne101 , converted your to a discussion. The answer is no, it won't affect Edge at all.

**Comment by ghost** [comment]
Ok, thanks for the help, it's much appreciated.

Thanks
Nex
